manage_user_page.php

```diff
@@ -362,7 +369,7 @@
 	<div id="manage-user-edit-div" class="form-inline pull-left">
 		<form id="manage-user-edit-form" method="get" action="manage_user_edit_page.php" class="form-inline"
 			<?php # CSRF protection not required here - form does not result in modifications ?>>
-			<label class="inline" for="username"><?php echo lang_get( 'search' ) ?></label>
+			<label class="inline" for="username"><?php echo lang_get( 'search' )?> <i class="fa fa-info-circle" title="<?php echo lang_get('user_search_hint')?>"></i> </label>
```

Comment:
I prefer placeholder as is works better with mobile devices, e.g. we use it for username on login page https://github.com/mantisbt/mantisbt/blob/master-2.10/login_page.php#L239

Reply:
OK, got. Will remember it. Do you need I change this as do a new Pull Request? please let me know

Reply:
> Do you need I change this as do a new Pull Request?

No, you can go on committing/pushing to this one.
As a final step, we can squash all your commits to a single one.